apps/cow-fi/components/SearchBar.tsx

```diff
@@ -213,25 +312,125 @@ export const SearchBar: React.FC<SearchBarProps> = ({ articles }) => {
   }, [isFocused, highlightedIndex, filteredArticles])
 
   useEffect(() => {
```

Comment:
This hook body is big enough, I think it deserves a separate file

Reply:
Probably not relevant anymore.